src/contracts/core/DelegationManager.sol

```diff
@@ -637,7 +652,7 @@ contract DelegationManager is
         address staker,
         address operator,
         IStrategy[] memory strategies,
-        uint256[] memory sharesToWithdraw,
+        WithdrawableShares[] memory sharesToWithdraw,
```

Comment:
withdrawableSharesToWithdraw

Reply:
doesn't "sharesToWithdraw" already kinda infer that they're withdrawable shares?

Reply:
ambigious between shares and withdrawableShares